migrations/versions/57b52b75739b_.py

```diff
@@ -0,0 +1,39 @@
+"""empty message
```

Comment:
Don't forget to add a message to your migrations with `-m` when running the migration step.